packages/maskbook/src/extension/options-page/DashboardDialogs/Feedback.tsx

```diff
@@ -43,7 +43,7 @@ export function DashboardFeedbackDialog(props: WrappedDialogProps) {
                             label={t('your_message')}
                             value={message}
                             onChange={(e) => setMessage(e.target.value)}
-                            variant="standard"
+                            variant="outlined"
```

Comment:
🤔 i found many changes like this. is this a rebase mistake?

Reply:
I found this change does not follow the original style of text boxes.